src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/WebRequest/WebRequestGetResponseIntegration.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Net;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.ExtensionMethods;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.WebRequest
+{
+    /// <summary>
+    /// CallTarget integration for WebRequest.GetResponse
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = WebRequestCommon.NetFrameworkAssembly,
+        TypeName = WebRequestCommon.HttpWebRequestTypeName,
+        MethodName = MethodName,
+        ReturnTypeName = WebRequestCommon.WebResponseTypeName,
+        ParameterTypeNames = new string[0],
+        MinimumVersion = WebRequestCommon.Major2,
```

Comment:
We don't need to instrument assembly version `2.0.0.0` here. Since we're doing CallTarget, only the `4.0.0.0` will be loaded. This was a workaround in CallSite where older .NET Framework applications were built against an older framework so their assembly reference was `2.0.0.0` and we had to make sure to instrument with that information

Reply:
Alright 👍